ecloud/data_source_appliance.go

```diff
@@ -0,0 +1,68 @@
+package ecloud
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/ukfast/sdk-go/pkg/connection"
+	ecloudservice "github.com/ukfast/sdk-go/pkg/service/ecloud"
+)
+
+func dataSourceAppliance() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceApplianceRead,
+
+		Schema: map[string]*schema.Schema{
+			"name": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"appliance_id": &schema.Schema{
```

Comment:
We don't need to reference `appliance_id` here, as this is a duplicate of the datasource ID

Reply:
removed